### PR TITLE
Feature/restyle share ctx myasset

### PR DESCRIPTION
**Pull request by arealclimber**
### DEVELOP

- [x] userContext
- [x] BalanceSection
- [x] InterestSection
- [x] PnLSection
- [x] /api/images/cfd/[cfdId].tsx

### CHECK LIST

- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component:  0
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/balance_section/balance_section.tsx [hunk 1]
```diff
@@ -17,9 +17,9 @@ const BalanceSection = () => {
   const userCtx = useContext(UserContext);
 
   /* ToDo: (20230420 - Julian) getMyAssets by currency */
-  const avblBalance = userCtx.getMyAssets('')?.balance.available ?? 0; //userCtx.balance?.available ?? 0;
-  const lockedBalance = userCtx.getMyAssets('')?.balance.locked ?? 0; //userCtx.balance?.locked ?? 0;
-  const totalBalance = avblBalance && lockedBalance ? avblBalance + lockedBalance : 0;
+  const avblBalance = userCtx.getMyAssets('')?.balance.available ?? DEFAULT_BALANCE; //userCtx.balance?.available ?? 0;
```
Comment (arealclimber) [inline src/components/balance_section/balance_section.tsx:20]: 只需要拿一次 `getMyAssets`

File: src/pages/api/images/cfd/[cfdId].tsx [hunk 2]
```diff
@@ -32,16 +40,28 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
...
+  const bg =
+    pnlPercent > 0
+      ? pnlPercent > SHARING_BG_IMG_THRESHOLD_PNL_PERCENT
+        ? 'big-profit@2x'
+        : 'small-profit@2x'
+      : pnlPercent < 0
+      ? pnlPercent < -SHARING_BG_IMG_THRESHOLD_PNL_PERCENT
+        ? 'big-loss@2x'
+        : 'small-loss@2x'
+      : null;
```
Comment (arealclimber) [inline src/pages/api/images/cfd/[cfdId].tsx:61]: - 不賺不賠就是小賠的背景圖

Comment (arealclimber) [reply src/pages/api/images/cfd/[cfdId].tsx:61]: - 相對應的，需注意PNL文字顏色、顯示的箭頭等等

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**